framework/src/node/processor/processor.ts

```diff
@@ -469,6 +455,44 @@ export class Processor {
 		};
 	}
 
+	// eslint-disable-next-line class-methods-use-this
+	private _createScopedStateStore(stateStore: StateStore, moduleName: string): ModuleStateStore {
+		return {
+			account: {
+				get: async <T>(key: Buffer): Promise<Account<T>> => {
```

Comment:
Not in this PR, but in general should we use more descriptive names for generics. While looking at the code at this particular deep level, we miss the context that this generic type is.